more/free-programming-cheatsheets.md

```diff
@@ -182,6 +183,11 @@
 * [Rust Language Cheat Sheet](https://cheats.rs) (HTML)
 
 
+### Solidity
+
+* [Solidity](https://github.com/manojpramesh/solidity-cheatsheet)
```

Comment:
:+1: Like in:

https://github.com/EbookFoundation/free-programming-books/blob/b4450227d923fcb1cd96b3b3831aecfe33d6e9b3/courses/free-courses-id.md#L42